src/pe_source/data/helpers/redact_pii.py

```diff
@@ -0,0 +1,560 @@
+    regex = re.compile("|".join(PA), re.IGNORECASE)
+    filth_cls = PA_DLFilth
+
+
+class VT_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for VT drivers licenses."""
+
+    type = "VT_drivers_license"
+
+
+class VT_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify VT drivers licenses."""
+
+    name = "VT_drivers_license"
+    regex = re.compile("|".join(VT), re.IGNORECASE)
+    filth_cls = VT_DLFilth
+
+
+class VA_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for VA drivers licenses."""
+
+    type = "VA_drivers_license"
+
+
+class VA_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify VA drivers licenses."""
+
+    name = "VA_drivers_license"
+    regex = re.compile("|".join(VA), re.IGNORECASE)
+    filth_cls = VA_DLFilth
+
+
+class WA_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for WA drivers licenses."""
+
+    type = "WA_drivers_license"
+
+
+class WA_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify WA drivers licenses."""
+
+    name = "WA_drivers_license"
+    regex = re.compile("|".join(WA), re.IGNORECASE)
+    filth_cls = WA_DLFilth
+
+
+class WV_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for WV drivers licenses."""
+
+    type = "WV_drivers_license"
+
+
+class WV_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify WV drivers licenses."""
+
+    name = "WV_drivers_license"
+    regex = re.compile("|".join(WV), re.IGNORECASE)
+    filth_cls = WV_DLFilth
+
+
+class WI_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for WI drivers licenses."""
+
+    type = "WI_drivers_license"
+
+
+class WI_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify WI drivers licenses."""
+
+    name = "WI_drivers_license"
+    regex = re.compile("|".join(WI), re.IGNORECASE)
+    filth_cls = WI_DLFilth
+
+
+class WY_DLFilth(scrubadub.filth.Filth):
+    """Create filth class for WY drivers licenses."""
+
+    type = "WY_drivers_license"
+
+
+class WY_DLDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify WY drivers licenses."""
+
+    name = "WY_drivers_license"
+    regex = re.compile("|".join(WY), re.IGNORECASE)
+    filth_cls = WY_DLFilth
+
+
+# Build a detector to find Social security numbers with no spaces
+class SSNFilth(scrubadub.filth.Filth):
+    """Create filth class for Social Security numbers."""
+
+    type = "no_space_social_security_number"
+
+
+class SSNDetector(scrubadub.detectors.RegexDetector):
+    """Create detector class to identify social security numbers."""
+
+    name = "no_space_ssn"
+    regex = re.compile(
+        r"(?:(?<=\s)|(?<=^))(social security number|Social Security No|Social Security #|social security number|social|Social|ssn)\W*(?!219099999|078051120)(?!666|000|9\d{2})\d{3}(?!00)\d{2}(?!0{4})\d{4}(?=$|\s)",
```

Comment:
Remove duplicated clauses:
```suggestion
        r"(?:(?<=\s)|(?<=^))(social security number|Social Security No|Social Security #|social|ssn)\W*(?!219099999|078051120)(?!666|000|9\d{2})\d{3}(?!00)\d{2}(?!0{4})\d{4}(?=$|\s)",
```
(`Social` is redundant because you already have `social` and you are ignoring case.)